solc/CommandLineInterface.cpp

```diff
@@ -664,34 +664,51 @@ bool CommandLineInterface::parseLibraryOption(string const& _input)
 	for (string const& lib: libraries)
 		if (!lib.empty())
 		{
-			//search for last colon in string as our binaries output placeholders in the form of file:Name
-			//so we need to search for the second `:` in the string
-			auto colon = lib.rfind(':');
-			if (colon == string::npos)
+			//search for equal sign or last colon in string as our binaries output placeholders in the form of file=Name or file:Name
+			//so we need to search for `=` or `:` in the string
+			auto separator = lib.rfind('=');
+			bool isSeparatorEqualSign = true;
+			if (separator == string::npos)
 			{
-				serr() << "Colon separator missing in library address specifier \"" << lib << "\"" << endl;
-				return false;
+				separator = lib.rfind(':');
+				if (separator == string::npos)
+				{
+					serr() << "Equal sign separator missing in library address specifier \"" << lib << "\"" << endl;
+					return false;
+				}
+				else
+					isSeparatorEqualSign = false; // separator is colon
 			}
+			else
+				solAssert(lib.rfind('=') == lib.find('='), "Only one equal sign \"=\" is allowed in the address string \"" + lib + "\".");
```

Comment:
Nice

Reply:
This is the only place where an option `--libraries "x==0x1235"` is caught, right? In that case, it should not be a `solAssert` but instaed an error message using `serr` like above.

Reply:
OK. Done